### ISSUE TITLE
Show permission change then in report nothing was changed

### ISSUE THREAD

**Issue by ShockwaveNN** [fedora-ruby/gem-compare#35]
1. Run `gem compare aws-partitions 1.607.0 1.608.0 -F`
2. In results will be lines like:
```
Changed:
              (!) New permissions: 100644 -> 100644
              (!) New permissions: 100644 -> 100644
              (!) New permissions: 100644 -> 100644
```

Both left and right side of -> are the same

Not sure if this should be hidden or left or right part of it - incorrect

----
Version info:
```
# gem list | grep compare
gem-compare (1.2.0)
```

**Comment by dentarg** [comment]
Yeah something went wrong with https://github.com/fedora-ruby/gem-compare/pull/30

Example: https://github.com/dentarg/actions-test/pull/22#issuecomment-1197303375

**Comment by strzibny** [comment]
I released `1.2.1` with the fix.

**Comment by ShockwaveNN** [comment]
Thanks, I can confirm that with `1.2.1` diff is clear with permissions
